api/BaseAudioContext.json

```diff
@@ -640,24 +640,11 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/BaseAudioContext/createPeriodicWave",
           "spec_url": "https://webaudio.github.io/web-audio-api/#dom-baseaudiocontext-createperiodicwave",
           "support": {
-            "chrome": [
-              {
-                "version_added": "59",
-                "notes": "Default values supported"
-              },
-              {
-                "version_added": "30"
-              }
-            ],
-            "chrome_android": [
-              {
-                "version_added": "59",
-                "notes": "Default values supported"
-              },
-              {
-                "version_added": "30"
-              }
-            ],
+            "chrome": {
+              "version_added": "30",
+              "notes": "Before Chrome 59, all arguments must be specified."
```

Comment:
Can you say in more detail what didn't work before Chrome 59? I've looked at the source and can't find the the optionality of arguments changed in 59.

Reply:
More research needed then :)